straxen/rundb.py

```diff
@@ -295,3 +273,30 @@ def run_metadata(self, run_id, projection=None):
         if self.reader_ini_name_is_mode:
             doc['mode'] = doc.get('reader', {}).get('ini', {}).get('name', '')
         return doc
+
+
+def get_mongo_url(hostname):
+    if hostname.endswith('xenon.local'):
+        username = straxen.get_secret('mongo_rdb_username')
+        password = straxen.get_secret('mongo_rdb_password')
+        url_base = 'xenon1t-daq:27017,old-gw:27017/admin'
```

Comment:
`old-gw` is dead (long live `old-gw`), probably best not to include it in the URI.

Reply:
Let's do some voodoo to reincarnate old-gw soon. But good point let me change it